components/meta-packages/history/history

```diff
@@ -529,6 +529,7 @@ desktop/administration/gnome-system-tools@2.30.0,5.11-2017.0.0.1
 desktop/administration/system-tools-backends@1.4.2,5.11-2017.0.0.3
 desktop/archive-manager/file-roller@2.32.2,5.11-2017.0.0.3
 desktop/avant-window-navigator@0.3.2.1,5.11-2017.0.0.3
+desktop/calculator/gcalctool@5.30.2-2018.0.0.0
```

Comment:
From my understanding you will have to increase the build number for entries here.
Thus, it should be
desktop/calculator/gcalctool@5.30.2-2018.0.0.1

Reply:
I think you are right, changed accordingly.